src/drivebase/OctocanumDrive.cpp

```diff
@@ -52,14 +51,10 @@ namespace ADBLib
 	{
 		if (mode == mecanum)
 		{
-			speeds[frontLeft] = x + y + r;
+			speeds[frontLeft] = (-x + y + r);

```

Comment:
These are not equivalent.
The old version adds x y and r and then makes the sum negative.
The new version takes the negative of x and then adds y and r
Ex: x = 10, y = 5, r=2
Old: 10 + 5 + 2 = 17 * -1 = -17
New: -10 + 5 + 2 = -3

I don't entirely understand how to program the mecanum wheels, so I'm not sure which is correct, but wanted to point out the difference.

Reply:
Oh my god, I can't believe I did that. I just fixed that issue by reverting the two relevant changes in that file.

Erm... in my defense, I did that in the early morning (just committed days later), while still recovering from jet lag? I don't know, worth a shot! Sorry for the math error!

Reply:
No problem, I wasn't sure if it was intentional or not, so wanted to point it out.
As you said, the team isn't currently using this library so it wouldn't have broken anything, at least this season.

The rest of the changes look really good.  I'm a big fan of correct indentation 👍